src/mqt/predictor/result.py

```diff
@@ -34,19 +34,23 @@ def __init__(
         if compiled_qc is not None:
             rew_fid = reward.expected_fidelity(compiled_qc, device)
             rew_crit_depth = reward.crit_depth(compiled_qc)
+            rew_succ_prob = reward.expected_success_probability(compiled_qc, device)
```

Comment:
Could we always refer to the estimated success probability by the same term? I think this makes the code more maintainable. Because currently `succ_prob`, `success_probability` and `estimated_success_probability` is used. 

I am aware that also for the expected fidelity it is currently not the case, but it should be ;)